docs/moonlight/overview.md

```
@@ -128,6 +128,7 @@ Effects, Modifiers, Layouts and drivers use emoji's to visualize their usage. Se
     * 🎨 Using palette
     * 🐙 WLED origin
     * 💫 MoonLight origin
+    * ⚡: FastLED origin
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)